roles/StackStorm.st2/tasks/packs.yml

```diff
@@ -0,0 +1,15 @@
+---
+- name: Get installed st2 packs (in json format)
+  command: /usr/bin/st2 pack list -j
```

Comment:
CI is failing here: https://travis-ci.org/StackStorm/ansible-st2/jobs/498299271#L1288 with the following:
```
fatal: [localhost]: FAILED! => {"changed": false, "cmd": ["/usr/bin/st2", "pack", "list", "-j"], "delta": "0:00:00.356562", "end": "2019-02-25 18:25:31.978348", "msg": "non-zero return code", "rc": 1, "start": "2019-02-25 18:25:31.621786", "stderr": "", "stderr_lines": [], "stdout": "ERROR: HTTPConnectionPool(host='127.0.0.1', port=9101): Max retries exceeded with url: /v1/packs?include_attributes=ref%2Cname%2Cdescription%2Cversion%2Cauthor (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7ffb2fd1e290>: Failed to establish a new connection: [Errno 111] Connection refused',))", "stdout_lines": ["ERROR: HTTPConnectionPool(host='127.0.0.1', port=9101): Max retries exceeded with url: /v1/packs?include_attributes=ref%2Cname%2Cdescription%2Cversion%2Cauthor (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7ffb2fd1e290>: Failed to establish a new connection: [Errno 111] Connection refused',))"]}
```
Which means StackStorm is not running/started yet at the moment we're trying to run pack install.

Sounds like this functionality needs to be moved to a later post-services start step.